include/oneapi/dpl/pstl/execution_defs.h

```diff
@@ -220,13 +220,7 @@ auto
 __check_subscript(...) -> typename ::std::decay<_R>::type::value_type;
 
 template <typename _R>
-struct __range_traits
-{
-    using __value_t = decltype(__check_subscript<_R>(0));
-};
-
-template <typename _R>
-using __value_t = typename __range_traits<_R>::__value_t;
+using __value_t = typename _R::range_value_t;
```

Comment:
Should the name be `__range_value_t`? When it's just `__value_t` is seems to wide.

Reply:
try keeping the minimal changes.